### PR TITLE
VMware image deployment documentation for Proxy

### PR DESCRIPTION
**Pull request by keichwa**
**For link to rendered output,** see https://github.com/SUSE/spacewalk/issues/25264

- https://bugzilla.suse.com/show_bug.cgi?id=1227852
- https://github.com/SUSE/spacewalk/issues/25264

### CODE REVIEW

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc [hunk 1]
```diff
@@ -153,6 +159,37 @@ transactional-update
...
++
+For example:
++
+----
+mgr-storage-proxy /dev/nvme1n1
+----
++
+[NOTE]
+====
+This command will create the persistent storage volumes at [path]``/var/lib/containers/storage/volumes``.
```
Comment (aaannz) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc:184]:
```suggestion
This command will move the persistent storage volumes at [path]``/var/lib/containers/storage/volumes`` to the specified storage device.
```

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc [hunk 2]
```diff
@@ -70,6 +74,8 @@ This table shows the hardware requirements for deploying {productname} Proxy.
 |===
 
 
+// FIXME Do we need the following note here?
+
 [NOTE]
 ====
 [path]``/var/lib/containers/storage/volumes/srv-www``
```
Comment (aaannz) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc:81]: As we already established at [proxy-deployment-uyuni.adoc?plain=1#L65](https://github.com/uyuni-project/uyuni-docs/blob/8601d3295eb22545f51569e904b7e9480227cf9b/modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-deployment-uyuni.adoc?plain=1#L65) this is wrong and should be changed.

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc [hunk 3]
```diff
@@ -114,13 +120,13 @@ image::enter-timezone.png[]
...
 
-. Once installation completes login as root.
+. When installation completes log in as root.
 
 . Proceed to the next section.
 
 
 
-== Register {sl-micro} and {productname} {productnumber}
+== Register {sl-micro} and {productname} {productnumber} Proxy
```
Comment (aaannz) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc:129]: This macro is not correct for 5.5. It's still called SLE Micro 5.5. See [download filenames](https://www.suse.com/download/sle-micro/)

Comment (keichwa) [reply modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc]: ok, using sle-micro now.

**Review by aaannz — APPROVED** [review]
LGTM, thanks

**Pull request merged**